src/main/scala/firrtl/passes/AnnotateValidMemConfigs.scala

```diff
@@ -249,43 +248,42 @@ class AnnotateValidMemConfigs(reader: Option[YamlFileReader]) extends Pass {
       sp: Option[SRAMCompiler] = None, 
       dp: Option[SRAMCompiler] = None) {
     def serialize = {
-      if (sp != None) sp.get.serialize
-      if (dp != None) dp.get.serialize
+      sp match {
```

Comment:
Isn't this just more lines of code for doing the same thing? I guess this is more of a stylistic preference, but I personally don't get why this is better.